piccolo/apps/schema/commands/generate.py

```diff
@@ -107,31 +114,23 @@ def is_foreign_key(self, column_name: str) -> bool:
                 return True
         return False
 
-    def get_foreign_key_constraint_name(self, column_name) -> str:
+    def get_foreign_key_constraint_name(self, column_name) -> ConstraintTable:
         for i in self.foreign_key_constraints:
             if i.column_name == column_name:
-                return i.constraint_name
+                return ConstraintTable(i.constraint_name, i.constraint_schema)
 
         raise ValueError("No matching constraint found")
 
 
-@dataclasses.dataclass
 class OutputSchema:
     """
     Represents the schema which will be printed out.
-
-    :param imports:
-        e.g. ["from piccolo.table import Table"]
-    :param warnings:
-        e.g. ["some_table.some_column unrecognised_type"]
-    :param tables:
-        e.g. ["class MyTable(Table): ..."]
-
     """
 
-    imports: t.List[str]
-    warnings: t.List[str]
-    tables: t.List[t.Type[Table]]
+    def __init__(self):
+        self.imports: t.List[str] = []
+        self.warnings: t.List[str] = []
+        self.tables: t.List[t.Type[Table]] = []
```

Comment:
It's possible to keep it as a dataclass, if you just want to add the defaults as lists:

```python
imports: t.List[str] = dataclasses.field(default_factory=list)
```